app/views/two_factor_authentication/sms_opt_in/error.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= image_tag(asset_url('alert/fail-x.svg'), alt: '', width: 54, class: 'margin-bottom-2') %>
```

Comment:
FYI to the ongoing work in #5890, we'll probably want a non-empty `alt` here.

Reply:
Thanks for that reminder, I'd seen that PR and forgot to put 2 + 2 together 🤦

Reply:
fa3ce4e525329b913c0c4438582955d24338c5c6